tests/TestOne/buildtool/bin/buildtool

```diff
@@ -0,0 +1,69 @@
+#!/bin/sh -e
+#
+# This is a very simple pjc-aware ocamlfind-based build tool.
+
+# Build
+
+DEPENDS=""
+for p in $cur_depends; do
```

Comment:
Same with `cur_target_dir` which should be `cur__target_dir`. The pjc docs were wrong so I'll update them now.